test/tests/mfem/kernels/curlcurl.i

```diff
@@ -3,7 +3,7 @@
 
 [Mesh]
   type = MFEMMesh
-  file = gold/small_fichera.mesh
+  file = ../../../../unit/data/small_fichera.mesh
```

Comment:
This will not work with installed builds (unless it's a symlink).

When we install the tests, you only get the data from the root (./test) in this case. However, if you create a symlink for

```
ln -s ../../../../unit/data/small_fichera.mesh small_fichera.mesh
```

and then use
```
file = small_fichera.mesh
```

during the install process we will follow the symlinks and duplicate the data

Reply:
Thanks for chiming in!

> This will not work with installed builds

That with conda?

> However, if you create a symlink

Okay, in that case, I'd perhaps create a `test/data/mfem` folder that symlinks `unit/data/mfem`? The latter doesn't exist yet, but I think it'd probably make sense to dump the meshes there instead of directly in `unit/data`?
This made me think of something crazier though, what do you think of symlinking directly the mfem meshes folder? For users who don't clone mfem, the symlinks would be broken, but if they are hidden in the folders I just mentioned, would this be problematic?

Reply:
I've moved all meshes to `test/tests/mfem/mesh` so it's the unit tests instead that rely on reading the meshes from outside their directory (which I presume is safer?). No symlinks, though that `mug.e` _is_ big...

PS: The bot complains when the repo size increases a lot but doesn't bring you cake when it reduces by 12MiB? Shame. xD

Reply:
Perfect, that'll do. The whole `test/tests` directory is copied, so any relative paths beneath that folder will work.

> PS: The bot complains when the repo size increases a lot but doesn't bring you cake when it reduces by 12MiB? Shame. xD

Well... it still exists in history, so it doesn't get removed 😢

Reply:
Well... it still exists in history, so it doesn't get removed 😢

> Indeed, `git clone --filter=blob:none` to the rescue perhaps.

Reply:
this is resolved?